mungegithub/mungers/milestone-maintainer.go

```diff
@@ -191,6 +365,8 @@ func ignoreObject(obj *github.MungeObject, activeMilestone string) bool {
 		return true
 	}
 
+	// TODO ignore closed issues
+
```

Comment:
Just check that `obj.Issue.State` is "open".

Reply:
Done.